utils.ts

```diff
@@ -274,6 +275,17 @@ export async function runInRepo(options: RunOptions & RepoOptions) {
 		} else {
 			overrides.vite = options.release
 		}
+		const viteManifest = await pacote.manifest(options.release)
```

Comment:
this should only be called if we need the manifest to determine the value for an override.

Reply:
ok pushed a new change, we'd have to call pacote in each if statement, but i think that's ok since it caches the results!